pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -171,6 +180,32 @@ func (c *Cluster) Start() error {
 		logger.Infof("mgr metrics service started")
 	}
 
+	// enable monitoring if `monitoring: enabled: true`
+	if c.monitoringSpec.Enabled {
+		if c.clusterInfo.CephVersion.IsAtLeastNautilus() {
+			logger.Debugf("starting monitoring deployment")
```

Comment:
I'd say that `Infof` would be acceptable here.

Reply:
ack.